_rest_routes.inc.php

```diff
@@ -886,14 +886,21 @@
         return $return;
     },
     "GET /fhir/Organization" => function (HttpRestRequest $request) {
-        RestConfig::authorization_check("admin", "users");
+        if (!$request->isPatientRequest()) {
+            RestConfig::authorization_check("admin", "users");
+        }
         $return = (new FhirOrganizationRestController())->getAll($request->getQueryParams());
         RestConfig::apiLog($return);
         return $return;
     },
     "GET /fhir/Organization/:id" => function ($id, HttpRestRequest $request) {
-        RestConfig::authorization_check("admin", "users");
-        $return = (new FhirOrganizationRestController())->getOne($id);
+        $patientUUID = null;
+        if (!$request->isPatientRequest()) {
+            RestConfig::authorization_check("admin", "users");
+            $patientUUID = $request->getPatientUUIDString();
```

Comment:
this is throwing me off a bit. Isn't this block for when not using a patient request?

Reply:
btw,
Highly recommend for folks to change github to dark theme while playing this youtube song at same time :) :
https://www.youtube.com/watch?v=X2LTL8KgKv8

![image](https://user-images.githubusercontent.com/278968/119242930-08ba4880-bb17-11eb-93ea-9ac92aa05882.png)

Reply:
looks like inferno needs to give in to the dark side

Reply:
and we can never forget this openemr classic :)
https://www.youtube.com/watch?v=xSmYAdiXb5M

Reply:
never :) mesmerized by the speed bag :boxing_glove: